internal/app/vodtester/vodtester_app.go

```diff
@@ -144,11 +145,107 @@ func (vt *vodTester) Start(fileUrl string, taskPollDuration time.Duration) (int,
 		}
 	}
 
+	filePath := "/tmp/uploadtester.mp4"
+	resp, err := http.Get(fileUrl)
+
+	if err != nil {
+		glog.Errorf("Error downloading fileUrl=%s err=%v", fileUrl, err)
+	}
+	defer resp.Body.Close()
+
+	out, err := os.Create(filePath)
+	if err != nil {
+		glog.Errorf("Error creating file filePath=%s err=%v", filePath, err)
+	}
+	defer out.Close()
+
+	assetName = fmt.Sprintf("vod_test_asset_%s_%s", hostName, time.Now().Format("2006-01-02T15:04:05Z07:00"))
+	requestUpload, err := vt.lapi.RequestUpload(assetName)
```

Comment:
It would be good to perform each of these tests in parallel as well, so we don't take the time of 3 uploads+processes for each continuous test iteration. WDYT?

I think the steps for that could be:
 - Create a helper for each of the tests being performed here (1 upload with url that we had before, 2 direct upload and 3 resumable upload)
 - Use a [WaitGroup](https://pkg.go.dev/sync#WaitGroup) or even better an [ErrGroup](https://pkg.go.dev/golang.org/x/sync/errgroup#Group) to perform them in parallel and return on errors from any of them.

You can ignore that `int` "error code" and just return the same for simplicity. I don't really think they are used for anything, just a tech debt we still have around here.